cls_Better_Access_Chart.cls

```diff
@@ -542,46 +545,42 @@ End Sub
 '### Public Methods
 
 Public Sub DrawChart()
-
-    'Declarations
-    Dim FileName As String
     
     CreateChartFile
-       
-    FileName = CurrentProject.Path & "\" & HtmlFileName
-    
-    Control.Object.Navigate FileName
+    Control.Object.Navigate getHTMLFileName
     
 End Sub
 
+Public Sub RemoveChartFile()

```

Comment:
What is the idea of this procedure.
At the moment I do not see where it is in use.

Reply:
We are creating a temporary file and we should clean it up,
either in Class_Terminate() but then we have to wait until the webcontrol has finished rendering the file
or in Form_Close calling RemoveChartFile

Reply:
btw: do your prefer to use  Scripting.FileSystemObject for deleting the file or declare the windows API call?

Reply:
> We are creating a temporary file and we should clean it up, either in Class_Terminate() but then we have to wait until the webcontrol has finished rendering the file or in Form_Close calling RemoveChartFile

Yes I agree with you. Deleting the temporary file at the end is a clean implementation.
I would prefer an approach that is implemented directly by the class module.
Additional code in another event of the form is not a clean approach to me. Here the programmer has to insert code in two places. I assume that the second one will be forgotten

Reply:
> btw: do your prefer to use Scripting.FileSystemObject for deleting the file or declare the windows API call?

I want to keep the class module small. Since I assume that the scripting object is always there, I would use it and save myself the API declaration.